CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Changelog
+Starting from v3.0.0, all notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
```suggestion
## [v3.0.0]
```

Reply:
(so merge this and #194 first, create the beta release and only then move those changes under a "v3.0.0 - 2022-04-##" section for creating the final release)